lib/simpleofflinecache.js

```diff
@@ -5,9 +5,32 @@ var cacheHelper = require('sw-cache-helper');
 var debug = 0 ? console.log.bind(console, '[SimpleOfflineCache]') :
  function(){};
 
-function SimpleOfflineCache(cacheName) {
+// Default Match options, not exposed.
```

Comment:
We have already a bug for that.
I think you are right, so far i would like to keep it there, but move it when we have a bit of time.